src/modules/coreinit/coreinit_exit.cpp

```diff
@@ -0,0 +1,24 @@
+#include "coreinit.h"
+#include "coreinit_exit.h"
+#include "utils/log.h"
+
+void
+_Exit()
```

Comment:
That would be an bad assumption though, the mangling is not guaranteed to generate a dissimilar name.